tests/framework/DataObjects/test_DataObject_metadata.xml

```diff
@@ -2,15 +2,16 @@
 <Simulation verbosity="debug">
   <TestInfo>
     <name>framework/DataObjects.dataObject_metadata</name>
-    <author>maljdan</author>
-    <created>2016-06-23</created>
+    <author>wangc</author>
+    <created>2015-09-02</created>
```

Comment:
Could not understand the changes made here?

Reply:
reverted